webexception/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
webexception/webexception.py,sha256=-oyqfwXGCW1yhf3Vyc0nwO9fFG02JTP1EMprTXRfEx4,740
webexception-1.0.3.dist-info/METADATA,sha256=ULfq6_fmZWdHAmVTc0fNf8C1htuGWqZG5MGJO27oZEs,6478
webexception-1.0.3.dist-info/WHEEL,sha256=7Z8_27uaHI_UZAc4Uox4PpBhQ9Y5_modZXWMxtUi4NU,88
webexception-1.0.3.dist-info/RECORD,,
